.github/workflows/sast.yml

```
@@ -44,6 +44,8 @@ jobs:
       uses: github/codeql-action/init@v3
       with:
         languages: ${{ matrix.language }}
+        disable-default-queries: true
```

actionlint (1.7.4)

input "disable-default-queries" is not defined in action "github/codeql-action/init@v3". available inputs are "build-mode", "config", "config-file", "db-location", "debug", "debug-artifact-name", "debug-database-name", "external-repository-token", "languages", "matrix", "packs", "queries", "ram", "registries", "setup-python-dependencies", "source-root", "threads", "token", "tools", "trap-caching"

(action)